test/x3/Jamfile

```diff
@@ -8,107 +8,99 @@
 }
 
 rule compile ( sources + : requirements * : target-name ? )
 {
-    target-name ?= $(sources[1]:D=:S=) ;
-    tests += [ testing.compile $(sources) : $(requirements) : $(target-name)-p3 ] ;
+    target-name ?= $(subproject-name)_$(sources[1]:D=:S=) ;
+    return [ testing.compile $(sources)
+           : $(requirements) : $(target-name) ] ;
 }
 
 rule compile-fail ( sources + : requirements * : target-name ? )
 {
-    target-name ?= $(sources[1]:D=:S=) ;
-    tests += [ testing.compile-fail $(sources) : $(requirements) : $(target-name)-p3 ] ;
+    target-name ?= $(subproject-name)_$(sources[1]:D=:S=) ;
+    return [ testing.compile-fail $(sources)
+           : $(requirements) : $(target-name) ] ;
 }
 
 ###############################################################################
 
-{
-    ###########################################################################
-    test-suite spirit_v3/qi :
-
-     [ run actions.cpp          : : : : x3_actions ]
-     [ run alternative.cpp      : : : : x3_alternative ]
-     [ run and_predicate.cpp    : : : : x3_and_predicate ]
-     [ run any_parser.cpp    : : : : x3_any_parser ]
-     [ run attr.cpp             : : : : x3_attr ]
-     [ run binary.cpp           : : : : x3_binary ]
-     [ run bool.cpp             : : : : x3_bool ]
-     [ run char1.cpp            : : : : x3_char1 ]
-     [ run char_class.cpp       : : : : x3_char_class ]
-     [ run container_support.cpp : : : : x3_container_support ]
-     [ run debug.cpp            : : : : x3_debug ]
-     [ run difference.cpp       : : : : x3_difference ]
-     [ run eoi.cpp              : : : : x3_eoi ]
-     [ run eol.cpp              : : : : x3_eol ]
-     [ run eps.cpp              : : : : x3_eps ]
-     [ run expect.cpp           : : : : x3_expect ]
-     [ run int1.cpp             : : : : x3_int1 ]
-     [ run kleene.cpp           : : : : x3_kleene ]
-     [ run lexeme.cpp           : : : : x3_lexeme ]
-     [ run lit1.cpp             : : : : x3_lit1 ]
-     [ run lit2.cpp             : : : : x3_lit2 ]
-     [ run list.cpp             : : : : x3_list ]
-     [ run matches.cpp          : : : : x3_matches ]
-     [ run no_case.cpp          : : : : x3_no_case ]
-     [ run no_skip.cpp          : : : : x3_no_skip ]
-     [ run not_predicate.cpp    : : : : x3_not_predicate ]
-     [ run optional.cpp         : : : : x3_optional ]
-     [ run plus.cpp             : : : : x3_plus ]
-     [ run with.cpp             : : : : x3_with ]
+run actions.cpp ;
+run alternative.cpp ;
+run and_predicate.cpp ;
+run any_parser.cpp ;
+run attr.cpp ;
+run binary.cpp ;
+run bool.cpp ;
+run char1.cpp ;
+run char_class.cpp ;
+run debug.cpp ;
+run difference.cpp ;
+run eoi.cpp ;
+run eol.cpp ;
+run eps.cpp ;
+run expect.cpp ;
+run int1.cpp ;
+run kleene.cpp ;
+run lexeme.cpp ;
+run lit1.cpp ;
+run lit2.cpp ;
+run list.cpp ;
+run matches.cpp ;
+run no_case.cpp ;
+run no_skip.cpp ;
+run not_predicate.cpp ;
+run optional.cpp ;
+run plus.cpp ;
+run with.cpp ;
```

Comment:
Where is container_support.cpp ??

Reply:
There are some other missed tests. I did not catch up all of them, it is a part of the other PR.

Reply:
Which other PR?
Is there a ticket for adding kicked tests again - containing a list of kicked test to add later again?

Big chance to forget/miss some of the kicked tests...